### ISSUE TITLE
Flyer concept

### ISSUE THREAD

**Issue by rub1e** [foundersandcoders/marketing#2]
I have a few issues with the flyer conceptually that I think were touched on in the biz meeting but not fully explored.

For me, I think the most important thing is to maintain the integrity of the FAC brand and ensure people don't have a confused image of us. I'm also concerned about encroachment on the work that dwyl do, so very keen to hear how @iteles and @nelsonic feel.

* First and foremost, what is the purpose of this flyer? To promote your workshop? To promote FAC generally? To have something we can leave behind at meetups? I think the answer is "To promote your workshop" - and I think at the moment it's trying to do all three and more. So, the first thing to do would be to actually make it about the workshop - include dates, times, topics, outcomes, who's it aimed at, who's running it, etc. etc.
 * Which also means you should think about whether you want to link to eventbrite or have your own website (or page on FAC.com if that's allowed) - because the eventbrite link will be out of date after the first workshop and give your flyers a short shelf-life  

* Second, I'm not 100% comfortable with putting FAC front and centre in this flyer. For me, this is a group of FAC students using the skills that FAC helped them develop to educate others - but it is **not** a FAC workshop. As it is, it strongly implies the opposite. Which also means the basic effect of the first side of the flyer is to back-door a strapline for FAC - and we defo wouldn't use learn/code/build  
 * This is also true of the colour palette (though god knows we need one) and the font. (And, to some extent, the non-capitalisation of the words, which *some* people would fight about; but the oxford comma is appreciated.)
 * For now, let's say you'll be called **Liberty KoalaX**
  
* Overleaf, following on from above, I'd link to your own website - [libertyKoalaX.ly](). In that first section you might want to then pare back on the outcomes on offer - I'm assuming the workshops that this flyer will be relevant to will be more centred on HTML/CSS than API design. 
  
* And then that final section can be about the wider FAC, just tweaked a bit. So if the whole flyer is about Liberty KoalaX, you could says something like:

> Liberty KoalaX is made up of graduates of Founders & Coders, a free coding bootcamp that's changing the world etcetcetc. FAC can support you and your project with a huge range of services, from  simple prototypes/MVPs, to full production-grade applications, and everything in between. Get in touch etc.

  * And this is also where I'd be v interested to hear from dwyl

Just some thoughts - feel free to disagree

**Comment by iteles** [comment]
Some good thoughts here @rub1e!

My thoughts on this are pretty simple:
1. I strongly agree that we need to clarify ourselves how F&C and dwyl are working together, what the differences are between our offerings and which pieces of work/training/etc fall under whose purview - I suggest any interested parties (in the first instance myself, @nelsonic and @sofer) sit down together in the first week of January and agree on something by the end of that week. This must be done before flyers are printed.
+ I'm not sure I totally agree with this, though I see your point. I think in this instance it'll be down to @sofer and the conversations that have been had with @RachBLondon and @sohilpandya already!

(Merry festivities!)

**Comment by rub1e** [comment]
Totally - and I don't mean to lose all sense of perspective about people trying to provide a great value coding workshop - after all, this is exactly what we're all about! For me it's just about the integrity of the FAC brand and the reputational capital we've built up - in lieu of an AQA/Edexcel-type Examination board for software development, there's a strong sense in which our brand is our graduates' certificate.

**Comment by sohilpandya** [comment]
@rub1e 
* The purpose of this flyer is not just our workshop, we were thinking that it would be used by former and future Faccers(is this official yet?) for promotion to bring in MVP's/DWYL projects, hence us wanting the feedback on how to improve it so that it covers every angle possible. (Business card a better way to do this?)
* In your opinion what categorises as an official **FAC** workshop?

**Comment by RachBLondon** [comment]
@rub1e yes from our conversations with Dan we were planning to run this as a workshop in FAC in a similar vein to the paid ones which previously ran at FAC (I know there was one for react and i think a responsive site one at the beginning of next year).  We definitely wouldn't want to confuse this with the 16 week full time course which FAC offers. My thoughts are that due to the difference in event (2 days vs 4 months) it's is unlikely that someone would be able to claim completing the full FAC off the back of this weekend, but perhaps it does need more clarity?

We thought building these workshops would be good way to fill the gap of reasonably priced workshops, and perhaps in the future, give students who were still on the 8 week core course the opportunity to earn a bit of extra cash before the start MVPs (that is if we manage to build up a structure that is sustainable).

We thought that flyers would be a good way to promote this, but also thought it made sense to put the MVP builds on them as well (since we have all been working hard to bring those in  as well). I agree that the is some crossover with DWYL, but hopefully @iteles and @nelsonic and @sofer will be able help give us some guidance on that in the new year.

The reason why we decided not to make it event specific, was that we were thinking the potentially in the future we would run additional workshops. We thought if we directed people from the flyers to the eventbrite organiser page (something we need to set up) then the flyers wouldn't have to be used for a specific weekend (which seemed rather costly for printing etc).

Also the copy was written just to get things started, I am a terrible dyslexic so if you do spot any typos, grammar errors please let me know.

Also Merry Xmas :-)

**Comment by sofer** [comment]
@sohilpandya @RachBLondon thank you so much for taking this workshop on. @rub1e thank you for raising this issue. @nelsonic @iteles and I have a lot to talk about next week. We may well want to review the message the flyer sends out, but I am very keen that we do so relatively quickly so that we do not interfere with Sohil and Rachel's preparations for the workshop.

**Comment by rub1e** [comment]
Good good. As I say, I'm mostly thinking from a branding/reptuation/identity perspective, and that's most pertinent to the promotion of wider FAC than the workshop - **so my strong suggestion is to treat these as separate**

* For the workshop, I'm really not sure that a flyer is the most efficient way to target relevant people. Given our meetup mailing list with c. 2000 members as well as twitter, I'd have thought we can reach far more people for free through digital than we can with paid-for printed flyers - and digital is obviously a lot more shareable, so our reach would be even greater. Even if we did some facebook advertising we could do better targetting.
 * Which reminds me - I think we need a Facebook presence...

* For the flyer promoting wider FAC activities, obviously we have to wait for the discussion with dwyl for specifics, but more generally I think there's lots of work to be done here - and I think we're going to have a marketing/publicity working group in the new year that can take this on:
 * We've had a few semi-profitable discussions around a strapline - e.g. https://github.com/foundersandcoders/oursite/issues/2 & https://github.com/foundersandcoders/oursite/issues/102, but not sure we were really happy with any of the ideas. I think it's incredibly important to get this right and settle on something we're not going to want to change for at least a year
 * The flyer as-is misses a huge aspect of FAC - we are not just any coding bootcamp and we are not just any great-value development agency - the most important part of FAC is the story, the mission, the values, who we are and why we are doing this, the lives we've changed, etc.  
 * And from a design point of view, I think it should be unified with our new website design. And as someone pointed out, we might want a different picture than three middle class white men. So, again, I think devoting some real time to this would be a good investment

**Comment by sofer** [comment]
@rub1e :+1: (and we are on facebook, although nobody is currently maintaining our presence https://www.facebook.com/foundersandcoders/)
